app/lib/actions/katello/repository_set/enable_repository.rb

```diff
@@ -2,6 +2,8 @@ module Actions
   module Katello
     module RepositorySet
       class EnableRepository < Actions::EntryAction
+        middleware.use Actions::Middleware::BackendServicesCheck
```

Comment:
yeah, i'll do this in the Actions::Pulp::Abstract rather than Abstract::Async::Task to handle things like create/update/delete